package.json

```diff
@@ -23,10 +23,12 @@
     "eslint-config-airbnb-base": "11.3.1",
     "eslint-plugin-import": "2.7.0",
     "express": "4.15.3",
+    "grpc": "^1.4.1",
     "humps": "2.0.1",
     "knex": "0.13.0",
     "nodemon": "1.11.0",
     "pg": "6.4.0",
+    "protobufjs": "^6.8.0",
```

Comment:
@Dave-Gallup please remove the `^` from this dependency we futzed with yesterday so as to lock in its version, thanks!